setting_up_firely_server/configuration/logsettings.rst

```diff
@@ -143,7 +143,7 @@ Settings for the Console sink:
 		* ``{Level}``: Level of the log, see the values in :ref:`configure_log_level`
 		* ``{MachineName}``: Name of the machine hosting the Firely Server instance. Especially useful when running multiple instances all logging to the same file.
 		* ``{RequestId}``: Unique id of the web request, useful to correlate log statements
-		* ``{Message}}``: Actual message being logged
+		* ``{Message:l}``: Actual message being logged, with formatting
```

Comment:
I'd add the possible formats ("The :l format specifier switches off quoting of strings, and :j uses JSON-style rendering for any embedded structured data.") and a link to the relevant Serilog docs (https://github.com/serilog/serilog/wiki/Formatting-Output#formatting-plain-text)